submit-web/src/components/SubmissionItem/ConsultationRecord/DocumentUploadSection.tsx

```diff
@@ -100,7 +100,7 @@ export const DocumentUploadSection = () => {
             color: EAOColors.ProponentDark,
           }}
         >
-          Accepted file types: pdf, doc, docx, xlsx, Max. file size: 250 MB.
+          Accepted file types: pdf, doc, docx, xlsx. Max. file size: 250 MB.
```

Comment:
is this true, can we check that it can only accept these file types and not others